### PR TITLE
Merge configs fix

### PR DESCRIPTION
**Pull request by renatoi**
@src-code 

mergeConfigs() should return an empty object if argument is not an array or if it's an empty array

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/atomizer.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ module.exports = {
      * @return {object} An atomizer config object
      */
     mergeConfigs: function (configs) {
-        return _.merge.apply(null, configs.concat(utils.handleMergeArrays));
+        return Array.isArray(configs) && configs.length > 0 ? _.merge.apply(null, configs.concat(utils.handleMergeArrays)) : {};
```
Comment (src-code) [inline src/atomizer.js]: Are we polyfilling this?  Only works in IE9+.  Otherwise we need to use lodash here, since this code could run client-side.

Comment (renatoi) [inline src/atomizer.js]: I think we should. We should use http://polyester-site.rc.staging.manhattan.corp.bf1.yahoo.com/

Comment (renatoi) [inline src/atomizer.js]: Yeah, nvm, using lodash now.

**Pull request merged**